### PR TITLE
Port stat panel from tg

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

HTML stat panel from tg

## Why It's Good For The Game

Better perf when alt clicking on a tile, or when admins go on MC tab

## Changelog
:cl:
add: HTML stat panel
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
THANK YOU!

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -297,6 +298,17 @@
 /obj/item/proc/on_found(mob/finder as mob)
 	return
 
+/obj/item/proc/remove_item_verbs(mob/user)
```
Comment (TiviPlus) [inline code/game/objects/items.dm:301]: autodoc

File: code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -881,3 +885,26 @@ GLOBAL_VAR_INIT(automute_on, null)
...
+			continue
+		if(verb_to_init.hidden)
+			continue
+		if(!istext(verb_to_init.category))
+			continue
+		verb_tabs |= verb_to_init.category
+		verblist[++verblist.len] = list(verb_to_init.category, verb_to_init.name)
+	src << output("[url_encode(json_encode(verb_tabs))];[url_encode(json_encode(verblist))]", "statbrowser:init_verbs")
+
+/client/proc/check_panel_loaded()
```
Comment (TiviPlus) [inline code/modules/client/client_procs.dm:907]: autodoc

File: interface/skin.dmf [hunk 3]
```diff
@@ -329,17 +327,17 @@ window "preferences_window"
 window "statwindow"
 	elem "statwindow"
 		type = MAIN
-		pos = 281,0
+		pos = 372,0
 		size = 640x480
```
Comment (TiviPlus) [inline interface/skin.dmf:331]: is this making the stat window bigger?

**Review by TiviPlus — COMMENTED** [review]
Is this tested?

**Comment by BraveMole** [comment]
Possible the last changes didn't fix the admin tickets, so this will need to be tm'd at very low pop (and much better if it's someone with keyholding, so it has all perms)

**Pull request closed without merging**